### PR TITLE
Add a note in README about older versions

### PR DESCRIPTION
**Pull request by kchibisov**
Unfortunatelly it's not clear to everyone that they are looking at the development documentation.

### CODE REVIEW

**Review by chrisduerr — COMMENTED** [review]
Will this actually help? I'm not sure people actually read the documentation before opening issues.

**Comment by kchibisov** [comment]
They somehow found out that the docs are in `man 5 alacritty`, so they clearly read it.

File: README.md [hunk 1]
```diff
@@ -55,6 +55,10 @@ For everyone else, the detailed instructions to install Alacritty can be found
 
 ## Configuration
 
+**Alacritty versions before 0.13.0 are using YAML configuration file format
+and all their settings were documented in the `alacritty.yml`. The example
+`alacritty.yml` file for each release could be found on [GitHub releases page](https://github.com/alacritty/alacritty/releases).**
```
Comment (chrisduerr) [inline README.md:60]:
```suggestion
**Alacritty versions before 0.13.0 are using YAML configuration files
and all their settings were documented in the `alacritty.yml`. The example
`alacritty.yml` file for each release can be found on [GitHub releases page](https://github.com/alacritty/alacritty/releases).**
```

**Review by chrisduerr — CHANGES_REQUESTED** [review]

**Review by chrisduerr — APPROVED** [review]

**Comment by kchibisov** [comment]
Let's see how it goes, at least we could reference it now.

**Pull request merged**